fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -907,6 +908,7 @@ def state_dict(self, *args: Any, **kwargs: Any) -> Any:
         """
         if torch.cuda.is_available():
             torch.cuda.synchronize()
+        is_uninitialized = self._is_root is None
```

Comment:
here, we don't touch the init state in case we are just accessing the state before the training.

Reply:
Can we add a comment as to why we need this?

Reply:
sure. added a pointer to the comment below.